.changeset/lucky-bees-care.md

```diff
@@ -0,0 +1,29 @@
+---
+"@comet/cms-admin": minor
+"@comet/admin": minor
+---
+
+Add support for `badgeContent` prop in `MenuItemRouterLink`.
```

Comment:
```suggestion
Add support for `badgeContent` prop in `MenuItemRouterLink`
```

Nit: The title of a changeset usually has no period at the end

Reply:
https://github.com/vivid-planet/comet/pull/2431/commits/06ad97a4ed589a249ea784bcea9379f8b8416596